package.json

```diff
@@ -139,6 +139,9 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "nodemon -e js,jsx --exec npm run test-client",
```

Comment:
There is already another `config` related issues: #5856. My proposed solution was to always generate on the fly config fly when executing tests. Otherwise it can be almost impossible to have app running using `development` env and tests running with watch using `test` env.